target/linux/rockchip/patches-6.6/040-v6.10-arm64-dts-rockchip-Add-Radxa-ROCK-3C.patch

```diff
@@ -0,0 +1,769 @@
+From ee219017ddb50be14c60d3cbe3e51ac0b2008d40 Mon Sep 17 00:00:00 2001
```

Comment:
Please number the patches starting from 00x for rk3568 and other earlier SoCs.

And also squash NanoPi commits, leave last one is good enough.